tests/Spout/Writer/XLSX/SheetTest.php

```diff
@@ -131,4 +162,18 @@ private function assertSheetNameEquals($expectedName, $fileName, $message = '')
 
         $this->assertContains("<sheet name=\"$expectedName\"", $xmlContents, $message);
     }
+
+    /**
+     * @param string $fileName
+     * @param string $message
+     * @return void
+     */
+    private function assertIsHiddenSheet( $fileName, $message = '')
```

Comment:
rename to `assertContainsHiddenSheet` + remove space between `(` and `$fileName`